src/luna/pathology/cli/slide_etl.py

```diff
@@ -274,6 +230,29 @@ def get_slide(self, url, project_name="", comment="") -> Slide:
     # def estimate_stain(self, slide):
 
 
+def __slide_etl(
```

Comment:
Is it standard to named overloaded functions with two underscores? I think it might be `_slide_etl` instead of `__slide_etl`

Reply:
The intention here was to create some private functions that are not intended for use by the end-user. AFAICT, the convention is to use one or two underscores and not include docstrings (just regular comments).

Reply:
Gotcha. I think we use just one underscore elsewhere in the codebase, so I think we should keep it consistent and just use one (`_` ) instead of two `__`)